egs/wsj/s5/utils/make_absolute.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
Also, please add a few lines of comment at the top of this file saying what it does: something like:
```
# This script is a replacement for the command readlink -f (which is not portable).
# It turns a pathname into an absolute pathname, including following soft links.
```
Also, please use lowercase variable names in the bash script, for compatibility with the
other bash scripts in Kaldi.